packages/ui/components/VisualEditor/SchemaProperty.tsx

```diff
@@ -120,7 +169,10 @@ const SchemaProperty: React.FC<SchemaPropertyProps> = ({
             <option value="number">Number</option>
             <option value="boolean">Boolean</option>
             <option value="object">Object</option>
-            <option value="array">Array</option>
+            <option value="array<string>">Array&lt;String&gt;</option>
+            <option value="array<number>">Array&lt;Number&gt;</option>
+            <option value="array<boolean>">Array&lt;Boolean&gt;</option>
+            <option value="array<object>">Array&lt;Object&gt;</option>
```

Comment:
I would suggest that we should not use this type of array as a multiple option. We should have an single array than  selecting its type. It will look cleaner.